Mathlib/Topology/ContinuousFunction/Bounded.lean

```diff
@@ -10,6 +10,8 @@ import Mathlib.Analysis.NormedSpace.OperatorNorm.Basic
 import Mathlib.Analysis.CStarAlgebra.Basic
 import Mathlib.Analysis.Normed.Operator.ContinuousLinearMap
 import Mathlib.Topology.Bornology.BoundedOperation
+import Mathlib.RingTheory.Congruence.Basic
+import Mathlib.RingTheory.TwoSidedIdeal.Basic
```

Comment:
This gives you the lattice structure on `TwoSidedIdeal`, so you don't need to coerce `⊤` to `Set`. But actually, the fact that we need to add these imports means that we should just start a new file.
```suggestion
import Mathlib.RingTheory.TwoSidedIdeal.Lattice
```

Reply:
Should I move the contents to a new file? Is it always so when one adds a new import?

Reply:
It's not *always* the case, but it can be a red flag. In this case, we're adding some new imports (although only 3 new files transitively) to a file that is *already* 1500 lines long, and these imports are only used sparingly in that file.

Yes, I would move the contents to a new file. Maybe `Topology.ContinuousFunction.BoundedCompactlySupported`

Reply:
OK, I created a new file.